### PR TITLE
[#1364] Add link to CQC provider info

### PR DESCRIPTION
**Pull request by garethrees**
Fixes https://github.com/mysociety/whatdotheyknow-theme/issues/1364

Couple of [open questions](https://github.com/mysociety/whatdotheyknow-theme/issues/1364#issuecomment-1235302380) but figured I'd push up as I've written the code now.

Screenshot 2022-09-02 at 10 58 51

### CODE REVIEW

**Comment by garethrees** [comment]
> Couple of https://github.com/mysociety/whatdotheyknow-theme/issues/1364#issuecomment-1235302380 but figured I'd push up as I've written the code now.

Basically resolved, so I think this is good to go.

File: lib/views/public_body/_more_info.html.erb [hunk 1]
```diff
@@ -62,6 +62,13 @@
   <% end %>
 <% end %>
 
+<% if public_body.has_tag?('cqc') %>
```
Comment (mdeuk) [inline lib/views/public_body/_more_info.html.erb:65]: Left a query on https://github.com/mysociety/whatdotheyknow-theme/issues/1364#issuecomment-1235318944 regarding this - it _**may**_ make sense to split into `cqcprovider` and `cqclocation` (as many have both, which may not be the same thing!).

Comment (mdeuk) [inline lib/views/public_body/_more_info.html.erb:65]: Left a query on https://github.com/mysociety/whatdotheyknow-theme/issues/1364#issuecomment-1235318944 regarding this - it _**may**_ make sense to split into `cqcprovider` and `cqclocation` (as many have both, which may not be the same thing!).

**Review by mdeuk — COMMENTED** [review]
Looks good - one query left as to whether or not we want to split the tag.

**Review by gbp — COMMENTED** [review]
Going to leave this un-merged until we switch to using the `cqcprovider` and `cqclocation` tags.

**Comment by garethrees** [comment]
Going to close this until we resolve https://github.com/mysociety/whatdotheyknow-theme/issues/1363. Can reopen once we make a decision.

**Pull request closed without merging**